NitroxPatcher/NitroxPatcher.csproj

```diff
@@ -51,10 +51,12 @@
     <Reference Include="System.Data.DataSetExtensions" />
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
+    <Reference Include="UnityEngine.UI, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" />
```

Comment:
You need to pass a `HintPath` with the correct path, otherwise this `Reference` will eventually fail if build order is changed (right now you are relying on another project that builds before `NitroxPatcher` to copy this file to the build directory). The other qualifiers are not necessary either, we need to build on the dll's that come with SN at all times, no matter their version etc.
```xml
<Reference Include="UnityEngine.UI">
  <HintPath>$(SubnauticaManaged)\UnityEngine.UI.dll</HintPath>
</Reference>
```
Put the following right below the `Reference` for `UnityEngine`, at line 48.